code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -11,6 +11,7 @@
 	volume = 50
 	flags = OPENCONTAINER
 	can_examine_reagents = 1
+	banned_reagents = list("pacid","sacid")

```

Comment:
This is just a list of the banned reagents. It is only called in the one of the functions and isn't called at all in this glass.dm file.. Infact I can edit that out as it does absolutely nothing.
Thanks for pointing it out